From de5ebb4c40e003f1ff12c214ea15e12c3264fbcb Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Tue, 9 Jul 2019 17:34:13 +0530
Subject: [PATCH 326/386] octeontx2-af: Update hardware workarounds for 95xx A1
 silicon

This patch applies workarounds of 95xx A0 to 95xx A1 silicon.

Change-Id: I28f7c81ff418c9dc6139ba0f235d3a8e153ed10e
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/12286
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
index 2c09586795d6..91b199b8b191 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
@@ -485,7 +485,7 @@ static inline bool is_rvu_95xx_A0(struct rvu *rvu)
 {
 	struct pci_dev *pdev = rvu->pdev;
 
-	return (pdev->revision == 0x10) &&
+	return ((pdev->revision == 0x10) || (pdev->revision == 0x11)) &&
 		(pdev->subsystem_device == PCI_SUBSYS_DEVID_95XX);
 }
 
-- 
2.17.1

